tests/library/browsercontext-fetch.spec.ts

```diff
@@ -797,11 +797,6 @@ it('should dispose when context closes', async function({ context, server }) {
   expect(error.message).toContain('Response has been disposed');
 });
 
-it('should throw on invalid first argument', async function({ context }) {
-  const error = await context.request.get({} as any).catch(e => e);
-  expect(error.message).toContain('First argument must be either URL string or Request');
```

Comment:
I see this error in the code, why remove the test?

Reply:
Because we have 1000 more places where we don't tolerate poor types?